fuzz/fuzz_targets/bitcoin/script_bytes_to_asm_fmt.rs

```diff
@@ -23,3 +23,30 @@ fn main() {
         });
     }
 }
+
+#[cfg(all(test, fuzzing))]
+mod tests {
+    fn extend_vec_from_hex(hex: &str, out: &mut Vec<u8>) {
```

Comment:
IIUC this could be literally `out.extend(HexIterator::new(hex).map(Result::unwrap))`

Reply:
Good call. This code comes from an ancient version of rust-lightning and long predates us having any hex code :).